source/features/show-asset-download-count.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {v4, escapeKey} from '../libs/api';
+import {cloudDownload} from '../libs/icons';
+import {getOwnerAndRepo, getRepoPath} from '../libs/utils';
+import {wrap} from '../libs/dom-utils';
+
+interface Asset {
+	name: string;
+	downloadCount: number;
+}
+interface Tag {
+	[key: string]: Asset[];
+}
+async function getAssetsForTag(tags: string[]): Promise<Tag> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const data = await v4(
+		'{' +
+			tags.map(tag =>
+				escapeKey(tag) + `: repository(owner: "${ownerName}", name: "${repoName}") { release(tagName:"${tag}") { releaseAssets(first: 100) { edges { node { name downloadCount } } } } }`
```

Comment:
Since these all belong to one repo, you can move this loop inside the `repository()` query. 

Also it needs to be more readable on multiple lines

Reply:
This hasn't been resolved. The query should look like:

```gql
repository(owner: "${ownerName}", name: "${repoName}") {
	${api.escapeKey(tag)}: release(tagName:"${tag}") {
		releaseAssets(first: 100) {
			nodes {
				name
				downloadCount
			}
		}
	}
	${api.escapeKey(tag)}: release(tagName:"${tag}") {
		releaseAssets(first: 100) {
			nodes {
				name
				downloadCount
			}
		}
	}
	...etc
}
```

Not the current:

```gql
repository(owner: "${ownerName}", name: "${repoName}") {
	${api.escapeKey(tag)}: release(tagName:"${tag}") {
		releaseAssets(first: 100) {
			nodes {
				name
				downloadCount
			}
		}
	}
}
repository(owner: "${ownerName}", name: "${repoName}") {
	${api.escapeKey(tag)}: release(tagName:"${tag}") {
		releaseAssets(first: 100) {
			nodes {
				name
				downloadCount
			}
		}
	}
}
...etc
```

Reply:
i.e. something like

```diff
  		'{' +
+ 			`repository(owner: "${ownerName}", name: "${repoName}") {
  				tags.map(tag =>
- 				`repository(owner: "${ownerName}", name: "${repoName}") {
```